core/bind/core_bind.cpp

```diff
@@ -30,6 +30,8 @@
 
 #include "core_bind.h"
 
+#include "core/method_bind_ext.gen.inc"
```

Comment:
This should be removed if you no longer need more than 5 parameters?

Reply:
Yeah if that's what we're going for.